### PR TITLE
updated form history button/page for admin users

### PR DESCRIPTION
**Pull request by granttebeau**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/866744728/pulses/1500412688)

Updates for the form history button/page for admin users. Changes the form history button to match the staff one, and allows the admin user to select from all schools, regardless of what country their account is associated with. Also updates the submit a form page for admins to allow them to select any school. 

## This PR

see above, pretty straightforward implementation. 

## Screenshots

 

Screen Shot 2021-08-09 at 12 44 44 PM

Screen Shot 2021-08-09 at 12 44 34 PM

## Verification Steps

 

npm start and log in with an admin user. Then, click on the form history button and see if all schools are being displayed. Click on one that has reports, and all reports for that school will show up

### CODE REVIEW

File: src/containers/home/index.tsx [hunk 1]
```diff
@@ -218,77 +217,40 @@ const Home: React.FC = () => {
             </Row>
           )}
           <Row gutter={[32, 0]} wrap>
-            {privilegeLevel === PrivilegeLevel.ADMIN ? (
```
Comment (maxsebso) [inline src/containers/home/index.tsx:221]: We may not want to change this if Admins are the only ones that can see Data Statistics. Or are we planning on making a separate button for that?

File: src/containers/pastSubmissionsSchools/PastSubmissionsSchools.tsx [hunk 2]
```diff
@@ -25,12 +23,15 @@ const PastSubmissionsSchools: React.FC = (props) => {
...
-  > = useSelector(
-    (state: C4CState) =>
-      state.pastSubmissionSchoolsState.pastSubmissionsSchools,
+  // const availableSchools: AsyncRequest<
+  //   PastSubmissionsSchoolsResponse,
+  //   any
+  // > = useSelector(
+  //   (state: C4CState) =>
+  //     state.pastSubmissionSchoolsState.pastSubmissionsSchools,
+  // );
```
Comment (maxsebso) [inline src/containers/pastSubmissionsSchools/PastSubmissionsSchools.tsx:32]: why is it still commented out?

**Review by maxsebso — CHANGES_REQUESTED** [review]
Just a few things but besides that LGTM

File: src/containers/home/index.tsx (hunk 1 above)
Comment (granttebeau) [reply src/containers/home/index.tsx:221]: We're making a separate button for that I think, this one is meant to just be a form history button for admin users so regular users should be able to see it as well

**Review by maxsebso — APPROVED** [review]
LGTM

File: src/containers/selectSchool/SelectSchool.tsx [hunk 3]
```diff
@@ -95,7 +100,9 @@ const SelectSchool: React.FC = () => {
                       >
                         {Array.from(
                           availableSchools.result.filter(
-                            (school) => school.country === userInfo.country,
+                            (school) =>
+                              privilegeLevel === PrivilegeLevel.ADMIN ||
```
Comment (rymaju) [inline src/containers/selectSchool/SelectSchool.tsx:104]: ⛏️ the priv level check really doesnt belong in the filter lambda, I would prefer something like:

```ts
privilegeLevel === PrivilegeLevel.ADMIN ? availableSchools.result :  availableSchools.result.filter((school) => school.country === userInfo.country)
```

**Review by rymaju — APPROVED** [review]
LGTM

**Pull request merged**